contracts/tokenomics/staking/RecapitalizationManager.sol

```diff
@@ -96,6 +104,14 @@ contract RecapitalizationManager is AccessControl, RewardTokenManager {
         _depositToken(pool, sid, tid, depositedToken);
 
         distributionBlock = block.number;
+
+        emit RecapitalizedPoolByRewards(
```

Comment:
Why did you add only one event for one method? Please add all other events also.